meraki/api/batch/__init__.py

```diff
@@ -13,16 +13,16 @@
 
 # Batch class
 class Batch:
-    def __init__(self):
+    def __init__(self, enable_kwarg_validation):
```

Comment:
This should not be needed. None of the other config kwargs receive this treatment and we should not start treating kwargs this way.